core/templates/pages/story-viewer-page/story-viewer-page.mainpage.html

```diff
@@ -77,6 +77,37 @@
         width: 100px;
       }
 
+      .lesson-icon-completed {
+        position: absolute;
+        left: 50px;
+        width: 40px;
+        height: 40px;
+        background-color: #296E5F;
+        border-radius: 25px;
+        transform: rotate(-120deg)
+      }
+
+      .lesson-icon-uncompleted {
+        position: absolute;
+        left: 50px;
+        width: 40px;
+        height: 40px;
+        background-color: white;
```

Comment:
Use equivalent HEX value (see [guidelines](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).

Reply:
Has this been addressed?

Reply:
Done.